spateo/plotting/static/pairwise.py

```diff
@@ -0,0 +1,185 @@
+"""
+Plots for pairwise comparison analyses between groups of samples and across different conditions (e.g. biological
+conditions or using different clustering parameters).
+
+These functions build on the Label class- see spateo.utils.Label for documentation
+"""
+from typing import List, Tuple, Union
+
+import matplotlib as mpl
+import matplotlib.patheffects as PathEffects
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy
+from anndata import AnnData
+from matplotlib.collections import PolyCollection
+from matplotlib.ticker import StrMethodFormatter
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from ...utils import Label, interlabel_connections
+
+plt.rcParams["figure.dpi"] = 300
```

Comment:
spateo has a default figure dpi in the st.configuration.py file, probably no need to include it here

Reply:
https://github.com/aristoteleo/spateo-release/blob/3606ba7dda342854781d6e51b9d6a165a67d4a7f/spateo/configuration.py#L505